evaluation/smatch_score.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
I think this file could be renamed to maybe arcs_evaluation or arcs_metrics? It's more than Smatch score